probe-rs/tests/README.md

```diff
@@ -6,5 +6,7 @@ The source code for the tests can be found at locations below. Please note that
 - `inlined_function` 
   - The source for this binary is unknown. //TODO: Consider re-writing tests against source code in `probe-rs-debugger-test`, and removing the `inlined_function` binary from this repo.
 - `probe-rs-debugger-tests`
-  - This binary was created using the `STM32H745ZITx` feature of the [probe-rs-debugger testing application](https://github.com/probe-rs/probe-rs-debugger-test)
+  - This binary was created using the `STM32H745ZITx` feature of the [probe-rs-debugger testing application](https://github.com/probe-rs/probe-rs-debugger-test), and specific file versions as below:
+    - `main.rs` : https://github.com/probe-rs/probe-rs-debugger-test/blob/565132e3002860b1535908fbe7bf717188345ee7/src/main.rs
+    - `Cargo.lock`: https://github.com/probe-rs/probe-rs-debugger-test/blob/259dce3a8677c8bfe8382328d142c9a8995a3128/Cargo.lock
```

Comment:
This seems to point to a different commit compared to the `main.rs` file. Just mentioning a single commit for all files is probably easier.

It would also be good to indicate if the binary was compiled in release or debug mode, maybe just mention the correct command?

Reply:
If I want to reproduce the binary, I want to be able to do something like this:

- git checkout <commit id>
- cargo build --target xxx --features xxx --locked

But right now this is not easily done, since there are two separate commits mentioned, and I first have to figure out the necessary arguments to cargo. The `[profile.dev]` section changes the settings for a debug build, but that doesn't mean that I can't just use `--release` to perform a release build, so it would be nice to clearly specify how the binary is built.

Reply:
Done